aws/external/config.go

```diff
@@ -100,6 +100,15 @@ func (cs Configs) ResolveAWSConfig(resolvers []AWSConfigResolver) (aws.Config, e
 	return cfg, nil
 }
 
+// ResolveConfig calls the provide function passing slice of configuration sources
+func (cs Configs) ResolveConfig(f func(configs []interface{}) error) error {
```

Comment:
Does `ResolveConfig` need to be a separate method from `ResolveAWSConfig?  They basically do the same thing, with the exception, `ResolveAWSConfig` takes multiple resolvers where `ResolveConfig` only takes one. Does it make sense for `ResolveAWSConfigs` to be updated to take variadic arguments, instead of adding a new method?